src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -18,6 +18,7 @@ public enum IamAction {
   // datarepo
   LIST_JOBS,
   DELETE_JOBS,
+  // CONFIGURE is only available to admin users so can be used to check for admin access to TDR
```

Comment:
This helps, thanks!!

If you wished, you could make the comment in the form of a docstring.  That would make it show up when you hover on references.